mods/cnc/rules/misc.yaml

```diff
@@ -22,8 +22,18 @@ CRATE:
 		Condition: cloak-crate-collected
 	GiveMcvCrateAction:
 		SelectionShares: 0
-		NoBaseSelectionShares: 120
+		NoBaseSelectionShares: 50
 		Units: mcv
+	ExplodeCrateAction:
```

Comment:
Note that this makes the Temple of Nod affected/destroyable by it's own nuke - which I consider a good thing. I'd check and add the same tag to the Ion Cannon if missing for consistency.